### ISSUE TITLE
Accessing entry parent with GraphQL API throws error

### ISSUE THREAD

**Issue by dmgawel** [statamic/cms#3853]
## Bug Description

When using GraphQL API to fetch a hierarchical collection, accessing `parent` of nested entry results in the following error: `Statamic\\GraphQL\\Types\\EntryType::buildName(): Argument #1 ($collection) must be of type Statamic\\Contracts\\Entries\\Collection, null given, called in /Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/GraphQL/Types/EntryInterface.php on line 79`.

## How to Reproduce

1. Create an orderable collection (or use a starter kit and enable ordering for any collection, i.e. `articles`).
2. Nest an entry under another.
3. Fetch entries from this collection using GraphQL and access a `parent` field (example query in Extra Detail section)

Alternatively, we could fetch a specific entry that we know has a linked parent (is nested), but it results in the same error.

## Extra Detail

<details>
  <summary>Example query</summary>
  
  ```javascript
   entries(collection: "articles") {
    data {
      title
      ... on Entry_Articles_Article {
        id
        parent {
          id
        }
      }
    }
  }
  ```
</details>

<details>
  <summary>Error message and trace</summary>
  
  ```javascript
{
  "errors": [
    {
      "debugMessage": "Statamic\\GraphQL\\Types\\EntryType::buildName(): Argument #1 ($collection) must be of type Statamic\\Contracts\\Entries\\Collection, null given, called in /Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/GraphQL/Types/EntryInterface.php on line 79",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },
      "locations": [
        {
          "line": 6,
          "column": 7
        }
      ],
      "path": [
        "entry",
        "parent"
      ],
      "trace": [
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/GraphQL/Types/EntryInterface.php",
          "line": 79,
          "call": "Statamic\\GraphQL\\Types\\EntryType::buildName(null, instance of Statamic\\Fields\\Blueprint)"
        },
        {
          "call": "Statamic\\GraphQL\\Types\\EntryInterface::resolveType(instance of Statamic\\Structures\\Page, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/rebing/graphql-laravel/src/Support/InterfaceType.php",
          "line": 24,
          "function": "call_user_func_array(array(2), array(3))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Type/Definition/InterfaceType.php",
          "line": 130,
          "call": "Rebing\\GraphQL\\Support\\InterfaceType::Rebing\\GraphQL\\Support\\{closure}(instance of Statamic\\Structures\\Page, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 940,
          "call": "GraphQL\\Type\\Definition\\InterfaceType::resolveType(instance of Statamic\\Structures\\Page, null, instance of GraphQL\\Type\\Definition\\ResolveInfo)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 789,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), instance of Statamic\\Structures\\Page)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), instance of Statamic\\Structures\\Page)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(2), instance of Statamic\\Structures\\Page)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Entry_Articles_Article, instance of Statamic\\Entries\\Entry, instance of ArrayObject(1), array(2))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1145,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Entry_Articles_Article, instance of Statamic\\Entries\\Entry, array(1), instance of ArrayObject(2))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1106,
          "call": "GraphQL\\Executor\\ReferenceExecutor::collectAndExecuteSubfields(GraphQLType: Entry_Articles_Article, instance of ArrayObject(1), array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 974,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeObjectValue(GraphQLType: Entry_Articles_Article, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 789,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeAbstractValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 654,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValue(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 557,
          "call": "GraphQL\\Executor\\ReferenceExecutor::completeValueCatchingError(GraphQLType: EntryInterface, instance of ArrayObject(1), instance of GraphQL\\Type\\Definition\\ResolveInfo, array(1), instance of Statamic\\Entries\\Entry)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 1195,
          "call": "GraphQL\\Executor\\ReferenceExecutor::resolveField(GraphQLType: Query, null, instance of ArrayObject(1), array(1))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 264,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeFields(GraphQLType: Query, null, array(0), instance of ArrayObject(1))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/ReferenceExecutor.php",
          "line": 215,
          "call": "GraphQL\\Executor\\ReferenceExecutor::executeOperation(instance of GraphQL\\Language\\AST\\OperationDefinitionNode, null)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/Executor/Executor.php",
          "line": 156,
          "call": "GraphQL\\Executor\\ReferenceExecutor::doExecute()"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 162,
          "call": "GraphQL\\Executor\\Executor::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, instance of GraphQL\\Language\\AST\\DocumentNode, null, null, null, 'MyQuery', null)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/webonyx/graphql-php/src/GraphQL.php",
          "line": 94,
          "call": "GraphQL\\GraphQL::promiseToExecute(instance of GraphQL\\Executor\\Promise\\Adapter\\SyncPromiseAdapter, instance of GraphQL\\Type\\Schema, 'query MyQuery {\n  entry(slug: \"nectar\") {\n    id\n    ... on Entry_Articles_Article {\n      id\n      parent {\n        slug\n      }\n    }\n  }\n}', null, null, null, 'MyQuery', null, null)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/rebing/graphql-laravel/src/GraphQL.php",
          "line": 142,
          "call": "GraphQL\\GraphQL::executeQuery(instance of GraphQL\\Type\\Schema, 'query MyQuery {\n  entry(slug: \"nectar\") {\n    id\n    ... on Entry_Articles_Article {\n      id\n      parent {\n        slug\n      }\n    }\n  }\n}', null, null, null, 'MyQuery', null)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/rebing/graphql-laravel/src/GraphQL.php",
          "line": 120,
          "call": "Rebing\\GraphQL\\GraphQL::queryAndReturnResult('query MyQuery {\n  entry(slug: \"nectar\") {\n    id\n    ... on Entry_Articles_Article {\n      id\n      parent {\n        slug\n      }\n    }\n  }\n}', null, array(3))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/rebing/graphql-laravel/src/GraphQLController.php",
          "line": 73,
          "call": "Rebing\\GraphQL\\GraphQL::query('query MyQuery {\n  entry(slug: \"nectar\") {\n    id\n    ... on Entry_Articles_Article {\n      id\n      parent {\n        slug\n      }\n    }\n  }\n}', null, array(3))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/rebing/graphql-laravel/src/GraphQLController.php",
          "line": 48,
          "call": "Rebing\\GraphQL\\GraphQLController::executeQuery('default', array(3))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Controller.php",
          "line": 54,
          "call": "Rebing\\GraphQL\\GraphQLController::query(instance of Illuminate\\Http\\Request, 'default')"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php",
          "line": 45,
          "call": "Illuminate\\Routing\\Controller::callAction('query', array(2))"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Route.php",
          "line": 254,
          "call": "Illuminate\\Routing\\ControllerDispatcher::dispatch(instance of Illuminate\\Routing\\Route, instance of Rebing\\GraphQL\\GraphQLController, 'query')"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Route.php",
          "line": 197,
          "call": "Illuminate\\Routing\\Route::runController()"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 695,
          "call": "Illuminate\\Routing\\Route::run()"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 128,
          "call": "Illuminate\\Routing\\Router::Illuminate\\Routing\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/GraphQL/Middleware/CacheResponse.php",
          "line": 18,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\GraphQL\\Middleware\\CacheResponse::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/RequireStatamicPro.php",
          "line": 17,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\RequireStatamicPro::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/SwapExceptionHandler.php",
          "line": 19,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\SwapExceptionHandler::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 103,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 697,
          "call": "Illuminate\\Pipeline\\Pipeline::then(instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 672,
          "call": "Illuminate\\Routing\\Router::runRouteWithinStack(instance of Illuminate\\Routing\\Route, instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 636,
          "call": "Illuminate\\Routing\\Router::runRoute(instance of Illuminate\\Http\\Request, instance of Illuminate\\Routing\\Route)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Routing/Router.php",
          "line": 625,
          "call": "Illuminate\\Routing\\Router::dispatchToRoute(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 166,
          "call": "Illuminate\\Routing\\Router::dispatch(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 128,
          "call": "Illuminate\\Foundation\\Http\\Kernel::Illuminate\\Foundation\\Http\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/DisableFloc.php",
          "line": 18,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\DisableFloc::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/CheckMultisite.php",
          "line": 14,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\CheckMultisite::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/CheckComposerJsonScripts.php",
          "line": 14,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\CheckComposerJsonScripts::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/statamic/cms/src/Http/Middleware/PoweredByHeader.php",
          "line": 19,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Statamic\\Http\\Middleware\\PoweredByHeader::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/barryvdh/laravel-debugbar/src/Middleware/InjectDebugbar.php",
          "line": 60,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Barryvdh\\Debugbar\\Middleware\\InjectDebugbar::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php",
          "line": 21,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php",
          "line": 31,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\ConvertEmptyStringsToNull::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php",
          "line": 21,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php",
          "line": 40,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\TrimStrings::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php",
          "line": 27,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php",
          "line": 86,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/fruitcake/laravel-cors/src/HandleCors.php",
          "line": 38,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Fruitcake\\Cors\\HandleCors::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/fideloper/proxy/src/TrustProxies.php",
          "line": 57,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 167,
          "call": "Fideloper\\Proxy\\TrustProxies::handle(instance of Illuminate\\Http\\Request, instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php",
          "line": 103,
          "call": "Illuminate\\Pipeline\\Pipeline::Illuminate\\Pipeline\\{closure}(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 141,
          "call": "Illuminate\\Pipeline\\Pipeline::then(instance of Closure)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php",
          "line": 110,
          "call": "Illuminate\\Foundation\\Http\\Kernel::sendRequestThroughRouter(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/Projects/statamic-parent/public/index.php",
          "line": 55,
          "call": "Illuminate\\Foundation\\Http\\Kernel::handle(instance of Illuminate\\Http\\Request)"
        },
        {
          "file": "/Users/dawid/.composer/vendor/laravel/valet/server.php",
          "line": 219,
          "function": "require('/Users/dawid/Projects/statamic-parent/public/index.php')"
        }
      ]
    }
  ],
  "data": {
    [...]
  }
}
  ```
  
</details>

## Environment

Statamic 3.1.23 Pro
Laravel 8.46.0
PHP 8.0.6
statamic/ssg 0.4.0

**Install method (choose one):**
- Starter kit (Cool Writings)

**Comment by bobevsky** [comment]
@jasonvarga Will this be included in a new statamic update, because running composer update didn't solve this for me, i am still having the same issue.

**Comment by duncanmcclean** [comment]
It will be fixed in the next Statamic 3.1 update - not sure when that'll be released.
